src/parser-postcss.js

```diff
@@ -2,12 +2,41 @@
 
 const createError = require("./parser-create-error");
 
+// Postcss won't parse correctly selectors with the "@prettier-placeholder",
+// so we change it to a placeholder it can parse and then change it back
+// before printing.
+function fixSelectorPlaceholder(root) {
+  if (root.nodes) {
+    root.nodes.forEach(node => {
+      if (node.value && node.value.indexOf("PRETTIER_PLACEHOLDER") > -1) {
+        node.value = node.value.replace(
+          /PRETTIER_PLACEHOLDER/g,
```

Comment:
What happens if this string is in the source?

Reply:
@azz it gets replaced by `@prettier-placeholder`

Maybe add a sanity check? Throws if the counts don’t match